### PR TITLE
Intake index shoot

### PR DESCRIPTION
**Pull request by noahlinton**
Zach created new command group, and we tuned Auto1-4piece based on this.

### CODE REVIEW

File: src/main/java/frc/robot/commands/AutoIndexerShootCommand.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        timer.reset();
+        hasNoteTimer.stop();
+        hasNoteTimer.reset();
+        hadNote = false;
+    }
+
+    // Called every time the scheduler runs while the command is scheduled.
+    @Override
+    public void execute() {
+        hasNoteTimer.start();
```
Comment (Ben-Alderson) [inline src/main/java/frc/robot/commands/AutoIndexerShootCommand.java:44]: This timer doesn't seem to do anything. But I suppose it's harmless.

File: src/main/java/frc/robot/commands/IntakeCommand.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+    // Called when the command is initially scheduled.
+    @Override
+    public void initialize() {
+    }
+
+    // Called every time the scheduler runs while the command is scheduled.
+    @Override
+    public void execute() {
+        if (!isFinished()) {
```
Comment (Ben-Alderson) [inline src/main/java/frc/robot/commands/IntakeCommand.java:28]: I don't understand this logic. execute should never run if isFinished is true. Also isFinished always returns false. Just a bit confusing.

**Review by Ben-Alderson — APPROVED** [review]
A few kinda weird things here, but I don't see any real bugs. Looks good enough to merge.

File: src/main/java/frc/robot/commands/AutoIndexerShootCommand.java (hunk 1 above)
Comment (noahlinton) [reply src/main/java/frc/robot/commands/AutoIndexerShootCommand.java:44]: oh yes, missed this. I will remove

File: src/main/java/frc/robot/commands/IntakeCommand.java (hunk 2 above)
Comment (noahlinton) [reply src/main/java/frc/robot/commands/IntakeCommand.java:28]: Oh this is strange. Do we know who wrote that?

**Pull request merged**